src/pages/en/guides/data-fetching.md

```diff
@@ -4,35 +4,39 @@ title: Data Fetching
 description: Learn how to fetch remote data with Astro using the fetch API.
 ---
 
-Astro components and pages can fetch remote data to help generate your pages. Astro provides two different tools to pages to help you do this: **fetch()** and **top-level await.**
+Astro pages can fetch remote data at build time to help generate your pages.
 
-## `fetch()`
+## Astro `fetch()`
```

Comment:
This is a nit pick but I kept reading this as `Astro.fetch()` which suggests a special API. Maybe something more directly equivalent to the framework components heading could also work?

```suggestion
## `fetch()` on Astro pages
```